apps/sms/test/unit/compose_test.js

```diff
@@ -1454,43 +1482,94 @@ suite('compose_test.js', function() {
     });
   });
 
-  suite('Attachment pre-send menu', function() {
+  // generate checks for image, video, audio, and unknown button texts
+  ['image', 'video', 'audio', 'others'].forEach(function(type) {
```

Comment:
same typo here or maybe it's not a typo and you were going to update l10n id? :)